src/constants.py

```diff
@@ -14,8 +14,7 @@
 CURRENT_MODULE_DIR = os.path.dirname(__file__)
 CONFIG_DEFAULTS_PATH = os.path.join(CURRENT_MODULE_DIR, "defaults/config.json")
 SCHEMA_DEFAULTS_PATH = os.path.join(CURRENT_MODULE_DIR, "defaults/schema.json")
-TEMPLATE_DEFAULTS_PATH = os.path.join(
-    CURRENT_MODULE_DIR, "defaults/template.json")
+TEMPLATE_DEFAULTS_PATH = os.path.join(CURRENT_MODULE_DIR, "defaults/template.json")
```

Comment:
Would `os.path.join(CURRENT_MODULE_DIR, "defaults","template.json")` make more sense, eradicating the OS dependency? Windows machines would expect `\` if I'm correct.